build/Builder.js

```diff
@@ -4,22 +4,22 @@ const sass = require('sass');
 const fs = require('fs');
 const path = require('path');
 
-class Builder{
+class Builder {
 	vars = {};
 
-	setVariable(name, value){
+	setVariable(name, value) {
 		this.vars[name] = value;
 		return this; //allow chaining
 	}
 
-	getVariables(){
-		const varString = Object.values(this.vars).map((k, v) => `$${k}: ${v};` ).join("\n");
+	getVariables() {
+		const varString = Object.entries(this.vars).map(arr => `$${arr[0]}: ${arr[1]};`).join("\n");
```

Comment:
This works, but it could be made cleaner with array destructuring. 

`map([k, v] =>... ` 

If wanted, happy to merge as is just thought I'd mention it.

Reply:
Good spot, didn't even cross my mind!